plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -20,8 +20,9 @@ import (
 )
 
 const (
-	reqSizeLimit   = 1024 * 1024
-	reqEventsLimit = 10000
+	reqSizeLimit    = 1024 * 1024
+	reqEventsLimit  = 10000
+	logWarnInterval = 1 * 24 * time.Hour
```

Comment:
nit: This should be more specific. `logWarnInterval` is too generic. Is this interval for all `log.Warn` messages? It's also not an interval. An interval determines how often something happens. This is more like a threshold.